code/datums/shuttles.dm

```diff
@@ -159,17 +130,6 @@
 		without protective goggles. Emitters spawn powered on, expect \
 		admin notices, they are harmless."
 
-/datum/map_template/shuttle/emergency/imfedupwiththisworld
-	suffix = "imfedupwiththisworld"
-	name = "Oh, Hi Daniel"
-	description = "How was space work today? \
-		Oh, pretty good. We got a new space station and the company will make a lot of money. \
-		What space station? \
-		I cannot tell you; it's space confidential. \
-		Aw, come space on. Why not? \
-		No, I can't. Anyway, how is your space roleplay life?"
-	credit_cost = -5000
```

Comment:
small amount of space and spawns with a gun you can grief players with which is what usually happens